kubernetes/local/gateway-httpd-deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    software.uncharted.terarium: gateway-httpd
+  name: gateway-httpd
+spec:
+  containers:
+    - image: docker-hub.uncharted.software/auth/httpd-openidc:dev-1.0.0
```

Comment:
our proxy should not be used in this project as it is OSS and meant for others to use as well

Reply:
I have lost all knowledge of which docker repo names mean what.  Should we use `docker.uncharted.software` instead?  I will change to that unless someone says otherwise.